docs/source/conf.py

```diff
@@ -316,3 +318,21 @@ def get_incollection_template(self, e):
 
 # display logo on top-left of html pages
 html_logo = "img/unitary_fund_logo.png"
+
+myst_update_mathjax = False
+
+nbsphinx_custom_formats = {
+    '.mystnb': ['jupytext.reads', {'fmt': 'mystnb'}],
+}
+nbsphinx_execute = 'always'
+
+nbsphinx_thumbnails = {
+    'examples/ibmq-backends': '_images/unitary_fund_logo.png',
+    'examples/hamiltonians': '_images/vqe-cirq-pauli-sum-mitigation-plot.png',
+    'examples/simple_landscape': '_images/simple_landscape_13_0.png',
+    'examples/maxcut-demo': '_images/maxcut-demo_48_1.png',
+    'examples/pyquil_demo': '_images/pyquil_demo_11_0.png',
```

Comment:
```suggestion
    'examples/pyquil_demo': '_images/unitary_fund_logo.png',
```
At the moment,  the pyquil example is not executed in readthedocs. This means that also the image is not generated (see here https://mitiq.readthedocs.io/en/crazy4pi314-467/examples/examples.html ).    Until we figure out how to run Rigetti QVMs in readthedocs, it is probably better to use default image.